pom.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>uk.gov.companieshouse</groupId>
+	<artifactId>common-web-java</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
```

Comment:
Need to check the process around release versions.    @BSaunders95  If this review is now complete, do we manually set this version to 1.0.0-rc1 or is there a Maven release plugin that does this for us automatically via a Jenkins job or something?

Reply:
Spoke with Bradley, the pom version is irrelevant, the versioning is done via tagging so no issue here - just set it to unversioned